tests/testutils/test_utils.go

```diff
@@ -404,3 +406,45 @@ func compress(content string) []byte {
 	_ = bw.Close()
 	return buf.Bytes()
 }
+
+func EnableManager(inst *instance.Instance) (err error, shouldRemoveUUID bool) {
+	if inst.UUID == "" {
+		uuid, err := uuid.NewV4()
+		if err != nil {
+			return fmt.Errorf("Could not enable test instance manager: %w", err), false
+		}
+		shouldRemoveUUID = true
+		inst.UUID = uuid.String()
+	}
+
+	config, ok := inst.SettingsContext()
+	if !ok {
+		return fmt.Errorf("Could not enable test instance manager: could not fetch test instance settings context"), shouldRemoveUUID
+	}
+	manager_url, ok := config["manager_url"].(string)
+	if !ok || manager_url == "" {
+		return fmt.Errorf("Could not enable test instance manager: manager_url config is required"), shouldRemoveUUID
+	}
+
+	flags := inst.FeatureFlags
+	if flags == nil {
+		flags = map[string]interface{}{}
```

Comment:
We also need `inst.FeatureFlags = flags`, no?

Reply:
Hum it seems to be working this way. Aren't maps actually pointers?